README.md

```diff
@@ -0,0 +1,31 @@
+<!--
+   - SPDX-FileCopyrightText: 2022 Tezos Commons
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TC
+   -->
+
+# Homebase Lite
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-brightgreen.svg)](https://opensource.org/licenses/MIT)
+
+Homebase Lite is a Tezos smart contract for a decentralized voting system, based
+on “snapshots” of the blockchain.
+
+## Build Instructions
+
+Run `make` to build everything.
+
+## For Contributors
+
+Please see [CONTRIBUTING.md](CONTRIBUTING.md) for more information.
+
+## About Serokell
+
+Homebase Lite is maintained and funded with ❤️ by [Serokell](https://serokell.io/).
+The names and logo for Serokell are trademark of Serokell OÜ.
```

Comment:
I don't think this is quite right.

Reply:
It isn't at all, I thought I had removed this section in favor of the `License` one, but apparently I left it behind by mistake.